src/rust/runtime/mod.rs

```diff
@@ -146,6 +146,16 @@ impl DemiRuntime {
         }
     }
 
+    /// Removes the background `coroutine` associated with `handle`. Since background coroutines do not return a result
+    /// there is no need to cast it.
+    pub fn remove_background_coroutine(&mut self, handle: &TaskHandle) {
+        // 1. Remove Task from scheduler.
+        let _: Box<dyn Task> = self
+            .scheduler
+            .remove(handle)
+            .expect("Removing task that does not exist (either was previously removed or never inserted");
+    }
```

Comment:
I understand that we panic here if and only if something is badly broken. It would be nice to have extra information just in case if that happens.

I understand that we have two ways for approaching this:
- Change the function signature to Fail, instead of panicking, and then dump the execution context in the caller's functions.
- Keep the function signature as it is, and print some information based on the `TaskHandle`.

I personally prefer the first option.